scripts/measures/lib/validate-change-requests.ts

```diff
@@ -118,57 +170,63 @@ const cost_validationSchema: JSONSchemaType<Cost_MeasuresChange> = {
     type: 'object',
     properties: {
-        ...baseValidationSchemaProperties,
-        nqfId: { type: 'string', nullable: true },
-        nationalQualityStrategyDomain: { type: 'string', nullable: true },
-        isHighPriority: { type: 'boolean', nullable: true },
-        isInverse: { type: 'boolean', nullable: true },
-        isRiskAdjusted: { type: 'boolean', nullable: true },
-        primarySteward: { type: 'string', nullable: true },
-        allowedVendors: { type: 'array', items: { type: 'string' }, nullable: true },
-        allowedPrograms: { type: 'array', items: { type: 'string', enum: Object.values(Constants.ALLOWED_PROGRAMS) }, nullable: true },
-        eMeasureId: { type: 'string', nullable: true },
-        nqfEMeasureId: { type: 'string', nullable: true },
+        ...baseQualitySchemaProperties,
         measureSets: { type: 'array', items: { type: 'string' }, nullable: true },
-        isRegistryMeasure: { type: 'boolean', nullable: true },
-        metricType: { type: 'string', enum: Constants.METRIC_TYPES, nullable: true },
-        submissionMethods: { type: 'array', items: { type: 'string' }, nullable: true },
-        overallAlgorithm: { type: 'string', enum: Constants.OVERALL_ALGORITHM, nullable: true },
-        clinicalGuidelineChanged: { type: 'array', items: { type: 'string', enum: [...new Set(Object.values(Constants.COLLECTION_TYPES))] }, nullable: true },
-        historic_benchmarks: { type: 'object', nullable: true },
-        icdImpacted: { type: 'array', items: { type: 'string', enum: [...new Set(Object.values(Constants.COLLECTION_TYPES))] }, nullable: true },
     },
     required: ['measureId', 'category'],
     additionalProperties: false,
 } as JSONSchemaType<Quality_MeasuresChange>;
 
-export function initValidation(type: measureType, requireAll: boolean) {
-    return ajv.compile(getSchema(type, requireAll))
+const qcdr_validationSchema: JSONSchemaType<QCDR_MeasuresChange> = {
+    type: 'object',
+    properties: {
+        ...baseQualitySchemaProperties,
+        allowedVendors: { type: 'array', items: { type: 'string' }, nullable: true },
+    },
+    required: ['measureId', 'category'],
+    additionalProperties: false,
+} as JSONSchemaType<QCDR_MeasuresChange>;
+
+export function initValidation(type: measureType, isNewMeasure: boolean = false) {
+    return ajv.compile(getSchema(type, isNewMeasure))
 }
 
-function createSchema(schema: any, requireAll: boolean) {
-    if (requireAll) {
-        return {
-            ...schema,
-            required: Object.keys(schema.properties),
+function createSchema(schema: any, isNewMeasure: boolean) {
+    // If it's a new measure, some fields are required beyond the category and measureId.
+    if (isNewMeasure) {
+        switch (typeof schema) {
+            case typeof quality_validationSchema:
+                return {
+                    ...schema,
+                    required: [...baseQualityRequiredFields, 'measureSets'],
```

Comment:
Overwrites the `required` field for new measures, since far more fields are required than for change requests.